db/CSV_Writers/machineRunner.js

```diff
@@ -0,0 +1,18 @@
+const { userProvisions: uP, reviewProvisions: rP } = require('./config/provisions.js');
+const client = require('./config/PGIndex.js');
+
+(async () => {
+  try {
+    await client.connect();
+    await [uP, rP].reduce(async (count, provisions) => {
+      await count;
+      await require('./csvMachine').call(provisions);
```

Comment:
in youre PR you left the comment:

> csvMachine & machineRunner.js (should these be combined??)

the question i would ask is: would either of these functions need to be imported independently of one another? they seem pretty coupled/i dont think you would use one without the other. might be good to have both defined in one file